### PR TITLE
[#148] style: fix style - Eric

### LINKED ISSUE #148: e0958902 - stricter styling checks

### PR DESCRIPTION
**Pull request by e0958902**
# Checklist for feature implementation and modification
Closes #148 
- [x] Added relevant milestone (under milestone tab on the right)
- Documentation
  - [x] JavaDoc for classes and methods

# Checklist for non-implementation related PR
Closes #148 
- [x] Added relevant milestone (under milestone tab on the right)
- Documentation (Contributions)
  - [x] Miscellaneous and team-based admin tasks (if any)
 
# Additional Information
Fixes styles in the following files:
1. [ListCommand](https://github.com/AY2324S2-CS2113T-T09-1/tp/commit/5d71af19756fa233be8f21503f31ce694d65f72a)
2. [ListCommandTest](https://github.com/AY2324S2-CS2113T-T09-1/tp/commit/2a471b1b26ca887928879897e8daa9810b2b6ed2)
ListCommandTest checks is intended to have some failing checks as the methods authored by @T0nyLin 
3. [ViewCommand](https://github.com/AY2324S2-CS2113T-T09-1/tp/commit/86852fa9d212994ab0ce57f9b409374c3155f0cf)
4. [ViewCommandTest](https://github.com/AY2324S2-CS2113T-T09-1/tp/commit/ed8d7a68854d4c6f8fa407676b2fc3bf9492948d)
5. [MedicationManagerTest](https://github.com/AY2324S2-CS2113T-T09-1/tp/commit/51ca5e5e0ddea309673823b515d3191beeea9fd4)
6. [Medication](https://github.com/AY2324S2-CS2113T-T09-1/tp/commit/50762aabcfc947217fa561d40eae0e3434f4de49)

### CODE REVIEW

File: src/test/java/meditracker/medication/MedicationManagerTest.java [hunk 1]
```diff
@@ -3,13 +3,24 @@
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
 
+/**
+ * This test file is to perform tests on the MedicationManager
+ */
+
```
Comment (annoy-o-mus) [inline src/test/java/meditracker/medication/MedicationManagerTest.java:9]: There should not be a blank line between the javaDoc and the start of class/method

File: src/test/java/meditracker/medication/MedicationManagerTest.java [hunk 2]
```diff
@@ -3,13 +3,24 @@
...
 public class MedicationManagerTest {
 
+    /**
+     * Resets the MedicationManager
+     *
+     * @throws InvocationTargetException when it fails to call the reset method
+     * @throws IllegalAccessException when the method can not access the specified class
+     * @throws NoSuchMethodException
+     */
+
```
Comment (annoy-o-mus) [inline src/test/java/meditracker/medication/MedicationManagerTest.java:19]: Shift this javadoc to after the start of the `//@@author` tag, and also remove the blank line

**Review by annoy-o-mus — CHANGES_REQUESTED** [review]
Great job. Just some minor changes requested.

File: src/test/java/meditracker/medication/MedicationManagerTest.java (hunk 1 above)
Comment (e0958902) [reply src/test/java/meditracker/medication/MedicationManagerTest.java]: Got it. Resolved in [ac8ad1e](https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/240/commits/ac8ad1eafd5b86008ec9a621ca11cfe1d6cbe33f).

File: src/test/java/meditracker/medication/MedicationManagerTest.java (hunk 2 above)
Comment (e0958902) [reply src/test/java/meditracker/medication/MedicationManagerTest.java]: Got it. Resolved in [9a106bf](https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/240/commits/9a106bf99990df8b1eb65f1b78a133c32ce486b8).

**Pull request merged**